manifests/params.pp

```diff
@@ -45,7 +45,7 @@
   #     can be set to the user field that is in the group group_field
   $oned_ldap_user_group_field = hiera('one::oned::ldap_user_group_field','undef'),
   # should we enable opennebula repos?
-  $one_repo_enable = hiera('one::enable_opennebula_repo', 'true' ),
+  $one_repo_enable = hiera('one::enable_opennebula_repo', true ),
```

Comment:
Thats right, but there is no reason as AFAIK why the fallback parameter, which is puppet language and which is there in case the user does not use hiera, should be quoted too. Doesn't the spec tests use the fixture files?